.github/workflows/bindings-nodejs.yml

```diff
@@ -126,6 +132,12 @@ jobs:
           cache-dependency-path: sdk/src/client/bindings/nodejs/yarn.lock
           node-version: 14.x
 
+      - name: Install Required Dependencies (Ubuntu)
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install libudev-dev libusb-1.0-0-dev
```

Comment:
Do we really need this just to check the format ?

Reply:
Not anymore, changed it to not run the install script